Source/Core/Core/ConfigManager.cpp

```diff
@@ -11,6 +11,7 @@
 #include <sstream>
 #include <variant>
 
+#include <Core/Config/MainSettings.h>
```

Comment:
Same here. (Place the include below `#include "Common/Config/Config.h"`.)

Reply:
Also, `#include "name"` for Dolphin headers, not `#include <name>`.